packages/fxa-settings/src/models/Account.ts

```diff
@@ -559,17 +559,24 @@ export class Account implements AccountData {
     service?: string,
     redirectTo?: string
   ): Promise<PasswordForgotSendCodePayload> {
-    let serviceName;
+    let options: {
+      service?: string;
+      resume?: string;
+      redirectTo?: string;
+    } = {
+      resume: 'e30=', // base64 json for {}
+    };
+
+    // Important! Only set service when it's Firefox Sync
     if (service && service === MozServices.FirefoxSync) {
-      serviceName = 'sync';
-    } else {
-      serviceName = service;
+      options.service = 'sync';
```

Comment:
Just because it's relevant I'll copy the previous diff that was here:

![image](https://github.com/mozilla/fxa/assets/13018240/bb2753ab-9c10-486a-9d25-c512ee2a1918)

Looking more closely here I think the intent here was to send up `service` as `clientId` when we did the OAuth flow, which `serviceName = service;` in the current diff would have done I believe. I just checked locally and it looks like with this change, the "continue to" card header will hit the default "continue to account settings" since it can't check the service name, which makes sense. Probably not a big deal, but also not ideal if using one of the RP products was the user's goal with the PW reset.
<img width="400" alt="image" src="https://github.com/mozilla/fxa/assets/13018240/aa02662b-64da-4ced-baa7-a9ebdc077bcc">

In our weekly yesterday we confirmed the client ID sent out in stage should be matching `Firefox Sync`, meaning, the expected link should have contained `service=sync` instead of `service=${clientId}`. I just tried again in staging, and I'm definitely seeing the client ID here.

**The `service === MozServices.FirefoxSync` check seems to be the real problem** which relates to FXA-8290 / FXA-6836 (and may or may not have been an issue in production?). I'll go ahead and submit review so you can check me here, I feel like `getServiceName` on the integration should be able to account for this but there's something weird going on.

Reply:
Adding some more thoughts, I noted "and may or may not have been an issue in production" in my previous comment because I wonder if it's as simple as it's not hitting this conditional because the name in staging doesn't exactly match our enum.

This may explain why @vbudhram is seeing that "unable to fetch client ID" error (which I can't seem to see), if for some reason we don't store sync IDs in that "client config" for us to fetch from because we expect Sync services to come through as `'sync'` or something. Also perhaps we aren't catching something properly too which could explain the blank screen, I'm not sure why we aren't seeing "invalid parameter" but maybe that would be taken care of in FXA-8106.

Reply:
Given this feed back. I think we should treat FXA-6826 as a separate issue. With your approval, I'll roll back this particular part of the PR.

Reply:
Given this feedback, let's address [FXA-8358](https://mozilla-hub.atlassian.net/browse/FXA-8358), in a seperate ticket.

Reply:
Works for me. Might be related, I just tried going through a subscription product flow on staging 123done and the service name is defaulting to "account settings"... not sure if we pull that from `clientInfo` or what.

Reply:
Going to resolve this conversation, the bug should be taken care of in #15841